File: C#/Pascal's Triangle II/Pascal's Triangle II.cs
public class Solution {
    public IList<int> GetRow(int rowIndex) {
        List<int> result=new List<int>(){1};
        if(rowIndex==0) return result;
        int mid=rowIndex/2,i;
        for(i=1;i<=mid;i++)
            result.Add(result[i-1]/i*(rowIndex-i+1));
        if(rowIndex%2==0) i--;
        while(i>=1)
            result.Add(result[--i]);
        return result;
    }
    
}

File: Python/Pascal's Triangle II/Pascal's Triangle II.py
class Solution:
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """

        def c(n, r): return self.fact(n)//(self.fact(r)*self.fact(n-r))
        return [c(rowIndex, r) for r in range(rowIndex+1)]

    def fact(self, n):
        if(n <= 1):
            return 1
        return self.fact(n-1)*n
